### ISSUE TITLE
Vector tiles don't always work in 2D and CV

### ISSUE THREAD

**Issue by lilleyse** [AnalyticalGraphicsInc/cesium#7985]
Vector tiles are not well supported in 2D and CV. They use an ECEF based RTC value that doesn't adjust when switching to 2D or CV. And unlike regular polylines we don't store 2D position for vector tile polylines.

One example where the tileset disappears when switching to Columbus View: [Sandcastle](https://cesiumjs.org/Cesium/Build/Apps/Sandcastle/#c=dVTfb9sgEP5XUF7iSi7Gv7BZ02pq+jJp0iol6l78Qm3SsGGwADvLpv7vw8FJm1RFlmyO+/juvrvzQDUYONsxDW6BZDuwZIb3LXw62IJ5fdgulbSUS6bnIfhXSeCWYYLVliv5TTa8plZp8AVsqDAs9A5cbtS9+nO0VvL16qaSlRwco6mZZI7QM8PD9sYf1bRlmrqzgxH6rQd6Z8u0drE8ajXw5hD2FHKtGbXsp9KiWXuf4I3RcsEMsxecsNO85ZYPzEDaNME7AfwrfVh7YDCl3WvhMppDGJ2eVcdqEz1QS6MzUPTkBFJ6eo2mRyX2wslooikc+MsoOQ9HbXyoR7tLpdm7FFtuGLRbJoNNLw9yB1fHCnhpYKcMHw/O67ek2rovKtMAp2WCCYYFTgqMUYJDkCCI44zkuMQkL3CeheC6TGGZ5qRM4hSXeUrIGNE7noZrX/BPia4RJH4VKckQKcoyBAgihJKkIEkZJyTNEcZ5nEx25ExxnhVlmmVZTHKUXJD23ads5xcgVJI0JmMiCSRZWqIYZbFjdrlk7JqMhOS0YpyTPL3g0vxlaz+jc92ljAkutQjfAg3P7hnvfr2CytVO71wR3+rnWlPpUxHtVqvdZLs5zciKyqamxgo2tuVaKfFM9X1vrcPPVztu6y2wCiyf3Dx+bAzf2q3S3Xatlkr07XNvxoEeNZtYZuFsYexesDuP+crbTmk79nfgetqythNumkz03Ne/XUPWxozIRXQELRo+AN7cVrOLX0Q1A7WgxriTTS/Eiv9l1exuETn/M5hQtOHy5cfAtKD70WUb3333RgjhInLbjyjrtXh3438).

**Comment by mramato** [comment]
This is true for b3dm tiles as well, isn't it?

**Comment by lilleyse** [comment]
Yeah there are a bunch of issues open for b3dm in 2D/CV too. Maybe everything should be consolidated into one issue.

https://github.com/AnalyticalGraphicsInc/cesium/issues/5112
https://github.com/AnalyticalGraphicsInc/cesium/issues/5729
https://github.com/AnalyticalGraphicsInc/cesium/issues/5955
https://github.com/AnalyticalGraphicsInc/cesium/issues/7915
